optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/stream/AbstractConstraintStreamScoreDirectorFactory.java

```diff
@@ -41,27 +38,16 @@
  * @see ConstraintStreamScoreDirector
  * @see ScoreDirectorFactory
  */
-public class ConstraintStreamScoreDirectorFactory<Solution_, Score_ extends Score<Score_>>
+public abstract class AbstractConstraintStreamScoreDirectorFactory<Solution_, Score_ extends Score<Score_>>
```

Comment:
So that I can only deal with ACN in code that is actually using Drools.

Reply:
ConstraintStreamScoreDirectorFactory is not public API. Users don't know it exists.
So why do we need to split it up in Abstract* and subclasses?

Reply:
ah, in the drools version we want to pass that boolean and in bavet it doesn't exist. ok, makes sense...